vcx/libvcx/src/api/vcx.rs

```diff
@@ -76,10 +76,14 @@ pub extern fn vcx_init (command_handle: u32,
         } else {
             match settings::process_config_file(&config_path) {
                 Err(e) => {
-                    println!("Invalid configuration specified: {}", e);
                     return error::INVALID_CONFIGURATION.code_num;
                 },
-                Ok(_) => (),
+                Ok(_) => {
+                    match settings::validate_payment_method() {
```

Comment:
As far as I  understand VCX should be able to work without any payment plugins. So, `CONFIG_PAYMENT_METHOD` should be an optional field. And it's better to move it validation to `settings/validate_config` function.

Reply:
> And it's better to move it validation to `settings/validate`_config function.

The problem there is that there is another init function that calls `validate _config` which does only wallet export/import logic and does not require a plugin to be loaded for it to function. This one on the other hand is called on App load so it made sense to check it only here -- I tried with he function in the `validate_config` and it broke half of the tests